api.swagger.yaml

```diff
@@ -6657,6 +6657,89 @@ paths:
         - SessionCookieHeader: []
         - APIKeyHeader: []
         - CognitoJwt: [ invotra/locations:write ]
+  '/locations/any/{location_id}':
+    parameters:
+      - description: The Invotra UUID of the location.
+        in: path
+        name: location_id
+        required: true
+        schema:
+          type: string
+          format: uuid
+    get:
+      tags:
+        - locations
+      responses:
+        '200':
+          description: Returns a single complete location object of any type.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/location-any-schema'
+        '400':
+          description: The supplied UUID was malformed.
+        '403':
+          description: The user does not have permission to view this location.
+        '404':
+          description: The UUID does not correspond to an existing location.
+      description: Get information about the location.
+      security:
+        - SessionCookieHeader: []
+        - APIKeyHeader: []
+        - CognitoJwt: [ invotra/locations:read ]
+  '/locations/{location_id}/members':
```

Comment:
Any risks for solr here / new data?

Some dbs will have a lot of users in certain locations

Reply:
This wont be a problem, no additional solr indexing, will just page through users index with a filter on location TID.